src/index.ts

```diff
@@ -119,3 +147,8 @@ export function pluginNodePolyfill(
 		},
 	};
 }
+
+export {
+	builtinMappingResolved,
+	resolvedPolyfillToModules,
```

Comment:
Should we add these two public methods to README?

Reply:
Yeah, updated.